src/HighLevelInterface/linprog.jl

```diff
@@ -88,8 +86,29 @@ function linprog(c::InputVector, A::AbstractMatrix, rowlb::InputVector, rowub::I
     end
 end
 
+function linprog(c::InputVector, A::AbstractMatrix, rowlb::InputVector, rowub::InputVector, lb::InputVector, ub::InputVector, solver::AbstractMathProgSolver = MathProgBase.defaultLPsolver, presolve::Bool)
```

Comment:
I'd suggest a different factoring of the code: 1) adding `presolve::Bool=false` as an extra argument to the `linprog` above, 2) putting the code below into another function called `presolve_linprog` 3) having `linprog` call `presolve_linprog` if `presolve` is true.